sdk/core/core-rest-pipeline/samples/v1/javascript/README.md

```diff
@@ -1,12 +1,3 @@
----
-page_type: sample
-languages:
-  - javascript
-products:
-  - azure
-urlFragment: core-rest-pipeline-javascript
----
-
```

Comment:
@witemple-msft This is the result of running dev tool samples publish after the packages are released. Are the changes here where the metadata is removed expected?